structures/occupancytypes.go

```diff
@@ -301,6 +302,34 @@ func res11swb() OccupancyTypeStochastic {
+	salinityydists[13], _ = statistics.Init([]float64{60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89}, []int64{66, 71, 62, 62, 55, 54, 63, 48, 50, 40, 38, 24, 43, 33, 32, 33, 32, 27, 30, 24, 121, 128, 119, 120, 117, 104, 86, 101, 111, 106})
+	salinityydists[14], _ = statistics.Init([]float64{61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91}, []int64{28, 65, 72, 58, 54, 50, 60, 60, 43, 49, 43, 31, 27, 39, 37, 29, 36, 28, 27, 29, 22, 122, 131, 116, 121, 116, 104, 85, 101, 111, 106})
+	salinityydists[15], _ = statistics.Init([]float64{63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93}, []int64{40, 76, 62, 62, 54, 48, 55, 63, 42, 51, 34, 28, 33, 36, 38, 31, 29, 36, 22, 28, 21, 120, 134, 113, 121, 116, 105, 84, 101, 111, 106})
+	salinityydists[16], _ = statistics.Init([]float64{65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95}, []int64{63, 70, 60, 64, 47, 51, 62, 50, 43, 49, 33, 29, 31, 38, 39, 29, 27, 36, 21, 32, 17, 120, 136, 110, 124, 112, 105, 84, 101, 111, 106})
+	salinityydists[17], _ = statistics.Init([]float64{67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98}, []int64{32, 66, 70, 55, 52, 50, 54, 62, 43, 48, 39, 37, 16, 42, 35, 35, 28, 25, 38, 21, 30, 16, 121, 136, 109, 123, 113, 103, 83, 102, 110, 106})
+	salinityydists[18], _ = statistics.Init([]float64{70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99}, []int64{66, 71, 62, 62, 55, 54, 63, 48, 50, 40, 38, 24, 43, 33, 32, 33, 32, 27, 30, 24, 14, 145, 138, 114, 141, 118, 92, 113, 124, 114})
+	salinityydists[19], _ = statistics.Init([]float64{73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99}, []int64{35, 81, 73, 71, 59, 61, 73, 49, 56, 37, 28, 53, 38, 37, 34, 41, 23, 34, 17, 29, 175, 159, 170, 143, 126, 145, 153})
+	salinityydists[20], _ = statistics.Init([]float64{75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99}, []int64{36, 92, 75, 74, 63, 71, 68, 60, 47, 35, 46, 44, 42, 32, 42, 32, 30, 19, 35, 193, 181, 189, 147, 172, 175})
+	var salinityStochastic = paireddata.UncertaintyPairedData{Xvals: salinityxs, Yvals: salinityydists}
+	//Depth,Salinity
+	sdf.DamageFunctions[hazards.Depth|hazards.Salinity] = salinityStochastic
+	//content function...defaults to default?

```

Comment:
or you could force the structure and content to the same curve - I have not followed up with FEMA - can you create a ticket to track my progress on that?